<!-- Carousel
================================================== -->
<div id="myCarousel" class="carousel slide" data-ride="carousel">
  <!-- Indicators -->
  <ol class="carousel-indicators">
    <li data-target="#myCarousel" data-slide-to="0" class="active"></li>
  </ol>
  <div class="carousel-inner">
    <div class="item active">
      <img src="images/anim_bg_cropped.gif" class="center-block img-responsive" style="width:1450px;height:600px; filter: brightness(40%)">
      <div class="container">
        <div class="carousel-caption">
          <h1 class="fit-head" style="color:white">KRSA</h1>
          <p class="fit-text" style="color:white">Network-based Prediction of Differential Kinase Activity from Kinome Array Data</p>
          <p>
            <a class="btn btn-lg btn-primary" onclick="$('li:eq(1) a').tab('show');" role="button">Get started now &raquo;</a>
          </p>
        </div>
      </div>
    </div>

  </div>
  <a class="left carousel-control" href="#myCarousel" data-slide="prev"><span class="glyphicon glyphicon-chevron-left"></span></a>
  <a class="right carousel-control" href="#myCarousel" data-slide="next"><span class="glyphicon glyphicon-chevron-right"></span></a>
</div>
<!-- /.carousel -->

<div class="container marketing">



  <!-- START THE FEATURETTES -->

  <hr class="featurette-divider">

  <div class="row featurette center-on-xs">
  <div class="col-md-5 col-md-push-7">
    <img src="images/workflow.png" style="width:600px;height:800px;">
  </div>
  <div class="col-md-7 col-md-pull-5">
    <h2>About</h2>
    <p class="lead"> <b><big>KRSA</big></b> is an R shiny application that automates many of the steps required to anlayze <a href="https://pamgene.com/technology/" target="_blank" rel="noopener noreferrer">PamChip</a> data, including peptide filtering, random sampling, heatmap generation, and kinase network generation.<br><br> This app makes analyzing kinome array datasets accessible and eliminates much of the human workload that the previous method required. More importantly, KRSA represents the results in a bigger biological context by visualizing altered kinome signaling networks instead of individual kinases. <br><br>By designing this application with the web-based Shiny platform in R, we have made KRSA with a graphical user interface that is interactive and customizable, and open to timely updates as well as future expansion. KRSA can be used by biologists and data scientists alike, with no knowledge of statistical software required. <br><br>This all-in-one tool is designed to move raw data from the kinome array through the processing pipeline to interaction network generation, creating downloadable figures and tables along the way to facilitate simple integration into publications and presentations.<br> </p>
  </div>
  </div>
  
  <hr class="featurette-divider">
  <hr class="featurette-divider">
  
  <div class="row featurette center-on-xs">
  <div>
    <h2 class="featurette-heading">Publications</h2>
    <hr class="featurette-divider">
    <p class="lead" style= "font-size: x-large; font-weight: bold"><a data-toggle="collapse" data-parent="#accordionOne" href="#collapseOne">Abnormalities of signal transduction networks in chronic schizophrenia </a></p>
    <p style="font-size: small; font-style: italic; text-decoration: underline">NPJ Schizophrenia</p>
    <p style = "font-size: small">Jennifer L McGuire, Erica A Depasquale, Adam J Funk, Sinead M O'Donnovan, Kathryn Hasselfeld, Shruti Marwaha, John H Hammond, Vahram Hartounian, James H Meador-Woodruff, Jarek Melle, Robert E McCullumsmith</p>
     <div class="panel-group transparent-panel-group" id="accordionOne">
        <div class="panel panel-default">
          <div id="collapseOne" class="panel-collapse collapse">
            <div class="panel-body text-left">
              <p>Schizophrenia is a serious neuropsychiatric disorder characterized by disruptions of brain cell metabolism, microstructure, and neurotransmission. All of these processes require coordination of multiple kinase-mediated signaling events. We hypothesize that imbalances in kinase activity propagate through an interconnected network of intracellular signaling with potential to simultaneously contribute to many or all of the observed deficits in schizophrenia. We established a workflow distinguishing schizophrenia-altered kinases in anterior cingulate cortex using a previously published kinome array data set. We compared schizophrenia-altered kinases to haloperidol-altered kinases, and identified systems, functions, and regulators predicted using pathway analyses. We used kinase inhibitors with the kinome array to test hypotheses about imbalance in signaling and conducted preliminary studies of kinase proteins, phosphoproteins, and activity for kinases of interest. We investigated schizophrenia-associated single nucleotide polymorphisms in one of these kinases, AKT, for genotype-dependent changes in AKT protein or activity. Kinome analyses identified new kinases as well as some previously implicated in schizophrenia. These results were not explained by chronic antipsychotic treatment. Kinases identified in our analyses aligned with cytoskeletal arrangement and molecular trafficking. Of the kinases we investigated further, AKT and (unexpectedly) JNK, showed the most dysregulation in the anterior cingulate cortex of schizophrenia subjects. Changes in kinase activity did not correspond to protein or phosphoprotein levels. We also show that AKT single nucleotide polymorphism rs1130214, previously associated with schizophrenia, influenced enzyme activity but not protein or phosphoprotein levels. Our data indicate subtle changes in kinase activity and regulation across an interlinked kinase network, suggesting signaling imbalances underlie the core symptoms of schizophrenia. <a href="https://pubmed.ncbi.nlm.nih.gov/28900113/" target="_blank" rel="noopener noreferrer" style="font-size: medium;">PMID: 28900113</a></p>

            </div>
          </div>
        </div>
      </div>
  </div>
  <hr class="featurette-divider">
  <div>
    <p class="lead" style= "font-size: x-large; font-weight: bold"><a data-toggle="collapse" data-parent="#accordionTwo" href="#collapseTwo">Kinase network dysregulation in a human induced pluripotent stem cell model of DISC1 schizophrenia</a></p>
    <p style="font-size: small; font-style: italic; text-decoration: underline">Molecular Omics</p>
    <p style = "font-size: small">Eduard Bentea, Erica A K Depasquale, Sinead M O'Donovan, Courtney R Sullivan, Micah Simmons, James H Meador-Woodruff, Ying Zhou, Chongchong Xu, Bing Bai, Junmin Peng, Hongjun Song, Guo-Li Ming, Jarek Meller, Zhexing Wen, Robert E McCullumsmith</p>
     <div class="panel-group transparent-panel-group" id="accordionTwo">
        <div class="panel panel-default">
          <div id="collapseTwo" class="panel-collapse collapse">
            <div class="panel-body text-left">
              <p>Protein kinases orchestrate signal transduction pathways involved in central nervous system functions ranging from neurodevelopment to synaptic transmission and plasticity. Abnormalities in kinase-mediated signaling are involved in the pathophysiology of neurological disorders, including neuropsychiatric disorders. Here, we expand on the hypothesis that kinase networks are dysregulated in schizophrenia. We investigated changes in serine/threonine kinase activity in cortical excitatory neurons differentiated from induced pluripotent stem cells (iPSCs) from a schizophrenia patient presenting with a 4 bp mutation in the disrupted in schizophrenia 1 (DISC1) gene and a corresponding control. Using kinome peptide arrays, we demonstrate large scale abnormalities in DISC1 cells, including a global depression of serine/threonine kinase activity, and changes in activity of kinases, including AMP-activated protein kinase (AMPK), extracellular signal-regulated kinases (ERK), and thousand-and-one amino acid (TAO) kinases. Using isogenic cell lines in which the DISC1 mutation is either introduced in the control cell line, or rescued in the schizophrenia cell line, we ascribe most of these changes to a direct effect of the presence of the DISC1 mutation. Investigating the gene expression signatures downstream of the DISC1 kinase network, and mapping them on perturbagen signatures obtained from the Library of Integrated Network-based Cellular Signatures (LINCS) database, allowed us to propose novel drug targets able to reverse the DISC1 kinase dysregulation gene expression signature. Altogether, our findings provide new insight into abnormalities of kinase networks in schizophrenia and suggest possible targets for disease intervention.<a href="https://pubmed.ncbi.nlm.nih.gov/31106784/" target="_blank" rel="noopener noreferrer" style="font-size: medium;">PMID: 31106784</a> </p>

            </div>
          </div>
        </div>
      </div>
  </div>
  <hr class="featurette-divider">
  <div>
    <p class="lead" style= "font-size: x-large; font-weight: bold"><a data-toggle="collapse" data-parent="#accordionThree" href="#collapseThree">Emerging Kinase Therapeutic Targets in Pancreatic Ductal Adenocarcinoma and Pancreatic Cancer Desmoplasia </a></p>
    <p style="font-size: small; font-style: italic; text-decoration: underline">International Journal of Molecular Sciences</p>
    <p style = "font-size: small">Justin F Creeden, Khaled Alganem, Ali S Imami, Nicholas D Henkel, F Charles Brunicardi, Shi-He Liu, Rammohan Shukla, Tushar Tomar, Faris Naji, Robert E McCullumsmith</p>
     <div class="panel-group transparent-panel-group" id="accordionThree">
        <div class="panel panel-default">
          <div id="collapseThree" class="panel-collapse collapse">
            <div class="panel-body text-left">
              <p>Kinase drug discovery represents an active area of therapeutic research, with previous pharmaceutical success improving patient outcomes across a wide variety of human diseases. In pancreatic ductal adenocarcinoma (PDAC), innovative pharmaceutical strategies such as kinase targeting have been unable to appreciably increase patient survival. This may be due, in part, to unchecked desmoplastic reactions to pancreatic tumors. Desmoplastic stroma enhances tumor development and progression while simultaneously restricting drug delivery to the tumor cells it protects. Emerging evidence indicates that many of the pathologic fibrotic processes directly or indirectly supporting desmoplasia may be driven by targetable protein tyrosine kinases such as Fyn-related kinase (FRK); B lymphoid kinase (BLK); hemopoietic cell kinase (HCK); ABL proto-oncogene 2 kinase (ABL2); discoidin domain receptor 1 kinase (DDR1); Lck/Yes-related novel kinase (LYN); ephrin receptor A8 kinase (EPHA8); FYN proto-oncogene kinase (FYN); lymphocyte cell-specific kinase (LCK); tec protein kinase (TEC). Herein, we review literature related to these kinases and posit signaling networks, mechanisms, and biochemical relationships by which this group may contribute to PDAC tumor growth and desmoplasia. <a href="https://pubmed.ncbi.nlm.nih.gov/33233470/" target="_blank" rel="noopener noreferrer" style="font-size: medium;">PMID: 33233470</a></p>

            </div>
          </div>
        </div>
      </div>
  </div>
  <hr class="featurette-divider">
  <div>
    <p class="lead" style= "font-size: x-large; font-weight: bold"><a data-toggle="collapse" data-parent="#accordionFour" href="#collapseFour">Neuronal impact of patient-specific aberrant NRXN1α splicing</a></p>
    <p style="font-size: small; font-style: italic; text-decoration: underline">Nature Genetics</p>
    <p style = "font-size: small">Erin Flaherty, Shijia Zhu, Natalie Barretto, Esther Cheng, P J Michael Deans, Michael B Fernando, Nadine Schrode, Nancy Francoeur, Alesia Antoine, Khaled Alganem, Madeline Halpern, Gintaras Deikus, Hardik Shah, Megan Fitzgerald, Ian Ladran, Peter Gochman, Judith Rapoport, Nadejda M Tsankova, Robert McCullumsmith, Gabriel E Hoffman, Robert Sebra, Gang Fang, Kristen J Brennand</p>
     <div class="panel-group transparent-panel-group" id="accordionFour">
        <div class="panel panel-default">
          <div id="collapseFour" class="panel-collapse collapse">
            <div class="panel-body text-left">
              <p>NRXN1 undergoes extensive alternative splicing, and non-recurrent heterozygous deletions in NRXN1 are strongly associated with neuropsychiatric disorders. We establish that human induced pluripotent stem cell (hiPSC)-derived neurons well represent the diversity of NRXN1α alternative splicing observed in the human brain, cataloguing 123 high-confidence in-frame human NRXN1α isoforms. Patient-derived NRXN1+/- hiPSC-neurons show a greater than twofold reduction in half of the wild-type NRXN1α isoforms and express dozens of novel isoforms from the mutant allele. Reduced neuronal activity in patient-derived NRXN1+/- hiPSC-neurons is ameliorated by overexpression of individual control isoforms in a genotype-dependent manner, whereas individual mutant isoforms decrease neuronal activity levels in control hiPSC-neurons. In a genotype-dependent manner, the phenotypic impact of patient-specific NRXN1+/- mutations can occur through a reduction in wild-type NRXN1α isoform levels as well as the presence of mutant NRXN1α isoforms. <a href="https://pubmed.ncbi.nlm.nih.gov/31784728/" target="_blank" rel="noopener noreferrer" style="font-size: medium;">PMID: 31784728</a></p>

            </div>
          </div>
        </div>
      </div>
  </div>
  </div>

  
  


</div>
<!-- /END THE FEATURETTES -->
